src/openrct2-cli/Cli.cpp

```diff
@@ -1,4 +1,4 @@
-#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
```

Comment:
Oh I'm absolutely not advocating holding this back for trivia like copyright years - more of a general question. We can go through and update all files that've been touched this year in one sweep some other time.